Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -65,21 +65,18 @@ class OpenSearchEngine: NSObject, NSCoding {
      * Returns the search URL for the given query.
      */
     func searchURLForQuery(_ query: String) -> URL? {
-        // Ecosia: Quick Search Shortcuts Experiment
-        /* 
-         If the search is performed via Ecosia, we want to follow the standard flow
-         which is now environment-dependant.
-         the function `getURLFromTemplate(::)` takes into account the ecosia search engine
-         which contains only the production URL.
-         Once this experiment is considered finished and the logic needs fallback,
-         leave only the following line of code for the whole function's body
+        /* Ecosia: Quick search shortcuts & Bing distribution Experiment
+         We have two experiments that potentially change the search url.
          
-         `URL.search(query: query)`
+         Once they are finished, we can move back to the following line:
+         `URL.ecosiaSearchWithQuery(query)`
          */
         if EngineShortcutsExperiment.isEnabled,
```

Comment:
Having the search shortcuts with a higher priority here still enables both experiments to run together since when searching from the top bar `fromTemplateURL.baseDomain != "ecosia.org"` will never be met - falling back to bing next.